sketch2Code.sketchplugin

```diff
@@ -103,9 +108,35 @@ function templateBinding(targetView)
 
           var layoutString = layoutDefinition(layer, layerName)
           [layouts addObject:"\t" + layoutString]
+          [allLayers addObject:layer]
+          //Set the layer hidden for exporting
+          [layer setIsVisible:false]
       }
     })
 
+    //Make the real export
+    enumerateArray(layersToSave, function(layer){
+
+      [layer setIsVisible:true]
+
+      var layerName = [layer name]
+      var layerClass = NSStringFromClass([layer class])
+      var viewClass = mappedClassNameForLayer(layer)
+      var viewDesc = viewDescriptors[viewClass]
+      exportImageToFile(baseDir + "/" +  layerName, layer, viewDesc["export"]["format"])  
+
+      [layer setIsVisible:false]
```

Comment:
We just setting it to visible before the export and after it setting back to hidden, to prevent it from showing on other layers.